README.md

```diff
@@ -12,7 +12,7 @@ A Client library for OAuth2 which uses JWT for the generation of access tokens.
               .subject("myapp ")
               .build();
 
-  TokenSource tokenSource = config.tokenSource();
+  TokenSource tokenSource = TokenSources.reusableTokenSource(new OkHttpTokenSource(config));
```

Comment:
this setup looks a llttle bit complicated to me.
```java
 options = DatastoreOptions.newBuilder()
             .setProjectId(projectId)
             .setCredentials(ServiceAccountCredentials.fromStream(new ByteArrayInputStream(serviceAccountKey.getBytes())))
             .setNamespace(namespace)
             .build();

Datastore ds = options.getService();
```
This how the Cloud platform Datastore service is been configured for secured access.

we've got JwtConfig, TokenSources, TokenSource, OkHttpTokenSource, BearerAuthenticationInterceptor, only for a setup.

can't we just have  new OkHttpClient.Builder(new BearerAuthenticationInterceptor(json)).build() or

new OkHttpClient.Builder(new ServiceAccountCredentials(json)).build()
where we can pass and the credentials separated by properties values new OkHttpClient.Builder(new ServiceAccountCredentials(clinetEmail, privatekey)).jwtKey("jwt key").build()

What do you think about this ?